pegasus/sites.v3/hourofcode.com/views/campaign_date/short_date.erb

```diff
@@ -0,0 +1 @@
+<%= campaign_date('short') %>
```

Comment:
I'd much prefer to update the existing https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/hourofcode.com/views/campaign_date/short.haml template. It can be converted from a haml to an erb if necessary